packages/commonwealth/server/bindings/bootstrap.ts

```diff
@@ -177,3 +187,22 @@ export function bootstrapContestRolloverLoop() {
     loop().catch(console.error);
   }, 1_000 * 60);
 }
+
+export function bootstrapCommunityIndexerLoop() {
```

Comment:
also getting to the point where we better build a generic worker scheduler that can trigger multiple policies at different times